specification/devopsinfrastructure/Microsoft.DevOpsInfrastructure.Management/main.tsp

```diff
@@ -229,13 +229,13 @@ union OsDiskStorageAccountType {
   string,
 
   @doc("Standard OS disk type.")
-  "Standard",
+  Standard: "Standard",
```

Comment:
would this affect the SDK generation with or without these?

Reply:
This is for rename